src/client/containers/main/variables.css

```diff
@@ -0,0 +1,21 @@
+:root {
+  /* Fonts */
+  --font-title: 'Orbitron', sans-serif;
+  --font-ordinary: 'Khand', sans-serif;
+
+  /* Icons */
+  --fontSize-ModesIcons: 25px;
+  --fontSize-ButtonsIcons: 18px;
+
+  /* Text */
+  --mainText-ordinary-white: #FFFFFF;
+  --fontSize-title: 1rem;
+  --fontSize-text: 0.8rem;
+
+  /* Color Elements */
+  --bodyBackground-dark-light: #142A48;
+  --headerBackground-dark-super: #131E2F;
+  --mainText-accent-blue: #05D8C5;
+  --accentElement-terquoise: rgb(105, 221, 230);
+  --rangeSlider-ocean: #0B85B2;
+}
```

Comment:
missed empty line at the end of the file.

Reply:
done)